lib/msal-core/src/ServerRequestParameters.ts

```diff
@@ -82,7 +82,7 @@ export class ServerRequestParameters {
      * @param request
      * @param serverAuthenticationRequest
      */
-    populateQueryParams(account: Account, request: AuthenticationParameters, adalIdTokenObject?: any): void {
+    populateQueryParams(account: Account, request: AuthenticationParameters, adalIdTokenObject?: any, silentCall?: boolean): void {
```

Comment:
Could `adalIdTokenObject` be changed to type `object`? That would help catch mistakenly passing `silentCall` as the third parameter.

Reply:
Done